Runtime/PostProcessing/CinemachineVolumeSettings.cs

```diff
@@ -99,6 +99,18 @@ public enum FocusTrackingMode
         [Tooltip("This profile will be applied whenever this virtual camera is live")]
         public VolumeProfile m_Profile;
 
+        /// <summary>
+        /// Global override for the layer used by CinemachineVolumes.
+        /// </summary>
+        [Tooltip("Global override for the layer used by CinemachineVolumes. ")]
+        public static LayerMask s_LayerMaskOverride;
+
+        /// <summary>
+        /// If true, enables global layer override.
+        /// </summary>
+        [Tooltip("If true, enables global layer override. ")]
+        public static bool s_LayerMaskOverrideEnabled;
```

Comment:
Why static?  Having it as a global setting does not respond to the needs.  If you read the forum post, the guy wants a per-camera setting.